recipe/meta.yaml

```diff
@@ -28,11 +28,13 @@ requirements:
     - openblas 0.2.19|0.2.19.*
     - tbb
     - cmake
+    - metis ==5.1.*
 
   run:
     - blas 1.1 {{ variant }}
     - openblas 0.2.19|0.2.19.*
     - tbb
+    - metis ==5.1.*
```

Comment:
FYI don't think the `==` is necessary. `metis 5.1.*` works too and is a bit nicer looking. 😉